lib/handlers/site-templates.ts

```diff
@@ -51,7 +51,7 @@ function partition<T>(array: T[], filter: (value: T) => boolean): [T[], T[]] {
 export function loadSiteTemplates(configDir: string) {
     const [meta, templates] = partition(
         fs
-            .readFileSync(configDir + '/site-templates.conf', 'utf8')
+            .readFileSync(`${configDir}/site-templates.conf`, 'utf8')
```

Comment:
note: should be changed to `path.join()` instead

Reply:
I will take care of the path.join stuff in a different PR. Wish to avoid changing any behavior with this pr